tower-http/src/cors/allow_private_network.rs

```diff
@@ -39,6 +39,10 @@ impl AllowPrivateNetwork {
         Self(AllowPrivateNetworkInner::Predicate(Arc::new(f)))
     }
 
+    #[allow(
```

Comment:
Hmmm, I wonder why I don't need these on tower-async. Are we running different versions of clippy or different configs?

Reply:
Not sure. I really wanna cut down on the number of feature flags we use anyway. A job for another day 😅